### PR TITLE
Update installation.mdx

### LINKED ISSUE #4879: add info about how to upgrade rasa to docs
Right now, the installation section tells you how to install, but not how to upgrade your installed version. we should add that info to the documentation. Inspired by [this forum post](https://forum.rasa.com/t/how-do-you-upgrade-rasa/21845)

### PR DESCRIPTION
**Pull request by saravananselvamohan**
Added Upgrading Rasa Package docs

**Proposed changes**:
- Added Docs for Upgrading Rasa Packages
- Fix #4879

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [ ✔️] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by sara-tagger** [comment]
Thanks for submitting a pull request 🚀 @erohmensing will take a look at it as soon as possible ✨

File: docs/docs/installation.mdx [hunk 1]
```diff
@@ -184,6 +184,18 @@ git clone https://github.com/RasaHQ/rasa.git
...
 poetry install
 ```
+## Upgrading Rasa Packages
+
+Upgrade an already installed Rasa packages to the latest version from PyPI:
+
+```bash
+pip install --upgrade rasa
+```
+To Download a specific version, use required PyPI package version number:
```
Comment (erohmensing) [inline docs/docs/installation.mdx:194]:
```suggestion

To download a specific version, specify the version number:
```

File: docs/docs/installation.mdx [hunk 2]
```diff
@@ -184,6 +184,18 @@ git clone https://github.com/RasaHQ/rasa.git
 cd rasa
 poetry install
 ```
+## Upgrading Rasa Packages
+
+Upgrade an already installed Rasa packages to the latest version from PyPI:
```
Comment (erohmensing) [inline docs/docs/installation.mdx:189]:
```suggestion
To upgrade your installed version of Rasa Open Source to the latest version from PyPI:
```

File: docs/docs/installation.mdx [hunk 3]
```diff
@@ -184,6 +184,18 @@ git clone https://github.com/RasaHQ/rasa.git
 cd rasa
 poetry install
 ```
+## Upgrading Rasa Packages
```
Comment (erohmensing) [inline docs/docs/installation.mdx:187]:
```suggestion
## Upgrading Versions
```

**Review by erohmensing — COMMENTED** [review]
Thanks for adding this info to the docs! Would you mind moving it to the bottom of the page as well, so that the installation requirements for the pipeline dependencies stay with the installation instructions?

**Comment by saravananselvamohan** [comment]
> Thanks for adding this info to the docs! Would you mind moving it to the bottom of the page as well, so that the installation requirements for the pipeline dependencies stay with the installation instructions?

@erohmensing . Thanks for the valuable suggestions. I feel I made the changes required.

**Review by erohmensing — APPROVED** [review]
Indeed, looks great! Thanks for the contribution!

**Pull request merged**